contributors/guide/community-expectations.md

```diff
@@ -16,53 +16,56 @@ that shapes the rest of the project.  It currently covers the expectations
 of conduct that govern all members of the community as well as the expectations
 around code review that govern all active contributors to Kubernetes.
 
-### Code of Conduct
+## Code review
 
-The most important expectation of the Kubernetes community is that all members
-abide by the Kubernetes [community code of conduct](../../governance.md#code-of-conduct).
-Only by respecting each other can we develop a productive, collaborative
-community.
-
-### Code review
-
-As a community we believe in the [value of code review for all contributions](collab.md).
+As a community we believe in the value of code review for all contributions.
 Code review increases both the quality and readability of our codebase, which
 in turn produces high quality software.
 
 However, the code review process can also introduce latency for contributors
 and additional work for reviewers that can frustrate both parties.
 
 Consequently, as a community we expect that all active participants in the
-community will also be active reviewers.
-
-We ask that active contributors to the project participate in the code review process
-in areas where that contributor has expertise. Active
-contributors are considered to be anyone who meets any of the following criteria:
-   * Sent more than two pull requests (PRs) in the previous one month, or more
-   than 20 PRs in the previous year.
-   * Filed more than three issues in the previous month, or more than 30 issues in
-   the previous 12 months.
-   * Commented on more than five pull requests in the previous month, or
-   more than 50 pull requests in the previous 12 months.
-   * Marked any PR as LGTM in the previous month.
-   * Have *collaborator* permissions in the Kubernetes github project.
-
-In addition to these community expectations, any community member who wants to
-be an active reviewer can also add their name to an *active reviewer* file
-(location tbd) which will make them an active reviewer for as long as they
-are included in the file.
-
-#### Expectations of reviewers: Review comments
+community will also be active reviewers. The 
+[community membership](../community-membership.md) outlines the responsibilities
+of the different contributor roles. 
+
+All changes must be code reviewed. For non-maintainers this is obvious, since
```

Comment:
This paragraph isn't entirely accurate. It doesn't mention approvers/OWNERS, and it feels very light on the requirement for reviews. We should at least mention that the OWNERS process should be respected.

Reply:
A bunch of this stuff is just duplicated from the PR doc, so I think in the interest of brevity I'm just going to cut it out of here and then link to that.